cmd/metrics-v2.go

```diff
@@ -82,16 +84,28 @@ func init() {
 		getNodeHealthMetrics(),
 		getLocalDriveStorageMetrics(),
 		getCacheMetrics(),
-		getHTTPMetrics(),
+		getHTTPMetrics(false),
 		getNetworkMetrics(),
 		getMinioVersionMetrics(),
 		getS3TTFBMetric(),
 		getNotificationMetrics(),
 		getDistLockMetrics(),
 	}
 
+	bucketMetricsGroups := []*MetricsGroup{
+		getBucketUsageMetrics(),
+		getHTTPMetrics(true),
+		getBucketTTFBMetric(),
+	}
+
+	bucketPeerMetricsGroups = []*MetricsGroup{
```

Comment:
Q: why do we need `bucketPeerMetricsGroups` as we already publish bucket peer metrics in [bucketCollector](https://github.com/minio/minio/pull/17663/files#diff-245d74540f439e91f424f4f2d44e1f3f9385197d33144c8fe67f58f4c2f7afd2R2902)? Also, we don't have a collector for `bucketPeerMetricsGroups`

Reply:
That is needed for Peer calls for remote nodes to collect. 

Look at how existing peerMetricsGroups work.